src/clientapi/index.js

```diff
@@ -129,7 +129,8 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   var spritesheets;
   try {
     if (GLOBAL.CACHE) {
-      spritesheets = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+      var spriteSheetCache = GLOBAL.CACHE['spritesheets/map.json'] || GLOBAL.CACHE['spritesheets\\map.json'];
```

Comment:
It's okay, I would have definitely preferred to use path.join here.
On Apr 4, 2016 1:40 AM, "Jishnu Mohan" <notifications@github.com> wrote:

> In src/clientapi/index.js
> <https://github.com/gameclosure/devkit-core/pull/50#discussion_r58343436>:
>
> > @@ -129,7 +129,8 @@ exports.ClientAPI = Class(lib.PubSub, function () {
> >    var spritesheets;
> >    try {
> >      if (GLOBAL.CACHE) {
> > -      spritesheets = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
> > +      var spriteSheetCache = GLOBAL.CACHE['spritesheets/map.json'] || GLOBAL.CACHE['spritesheets\\map.json'];
>
> sorry. my bad. I thought this is was a node script.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gameclosure/devkit-core/pull/50/files/f7ffa625fc2fdd46136e5b18de76ba0f8abccd74#r58343436>
>